global-files/.github/labels.yml

```diff
@@ -86,6 +86,30 @@
   description: "Work on Test Runners"
   color: "ffffff"
 
+# The `x:rep/<value>` labels describe the amount of reputation to award
+# 
```

Comment:
Trailing whitespace on this line, FYI.

Reply:
Do you want to add that glorious configlet CI to this repo as well? :D

Reply:
OK, I can make a PR. 

The whitespace linting actions I found in the GitHub Marketplace seemed to be incomplete or have some flaw, so it was easier to just write:
https://github.com/exercism/configlet/blob/146d41c184b5/.github/bin/lint-whitespace

We could also consider adding it as an org-wide workflow. But this repo seems like a good first step.

Reply:
Yeah, I think for now just having it here would help.

Reply:
> OK, I can make a PR.

Created #205